src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestDot.java

```diff
@@ -27,7 +35,7 @@ public static void foo(int i, String s) throws FooExceptionForTestDot, BarExcept
 
     public void goodCaseForTestDot(int... aFoo)throws FooExceptionForTestDot, BarExceptionForTestDot
     {
-        String s = "ffffooooString";
+        String s = "ffffooooString"; // violation
```

Comment:
Violation is on line 38. Can't place comment.

Reply:
@shashwatj07 , please keep reference to all unfortunate cases at https://github.com/checkstyle/checkstyle/issues/10378